docs/content/en/configuration/notifications/smtp.md

```diff
@@ -132,6 +133,13 @@ to leave this as is, but you can customize it if you have issues or you desire t
 For security reasons the default settings for Authelia require the SMTP connection is encrypted by TLS. See [security]
 for more information. This option disables this measure (not recommended).
 
+### disable_starttls
+
+{{< confkey type="boolean" default="false" required="no" >}}
+
+For security reasons the default settings for Authelia require the SMTP connection do use STARTTLS when is available. See [security]
```

Comment:
@james-d-elliott ,  i need suggestion for this text

Reply:
Left a review! LGTM btw.